### ISSUE TITLE
internal function names

### ISSUE THREAD

**Issue by choisant** [pglpm/inferno#5]
Many of the internal functions, for instance in tplotfunctions.R, have names that cause confusion as they are not easy to identify as project specific functions.
Examples: pdff
Can we find a better naming convention?

**Comment by pglpm** [comment]
True. These plotting functions are all private functions that I wrote for my own personal use with R in general (not happy with ggplot and similar).

So one first question is whether we want to include custom-made plotting functions, or whether we want to leave the plotting problem to the user. I personally would prefer the first alternative, especially thinking about cases of users who might use R solely to run this package.

With the first alternative, then yes better naming should be done!
